### PR TITLE
Fix streamer not working on meteor restart

### PR DESCRIPTION
**Pull request by jfsiebel**
The problem was caused when meteor restarted without reloading the page, the ddp subscription called a `onStop` event that set all handlers to `undefined` after we set the handlers on the streamer init function.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/cursor/service.js [hunk 1]
```diff
@@ -35,13 +35,29 @@ export function publishCursorUpdate(payload) {
...
+  /**
+  * We create a promise to add the handlers after a ddp subscription stop.
+  * The problem was caused because we add handlers to stream before the onStop event happens,
+  * which set the handlers to undefined.
+  */
+  cursorStreamListener = new Meteor.Streamer(`cursor-${Auth.meetingID}`, { retransmit: false });
 
+  const startStreamHandlersPromise = new Promise((resolve) => {
+    const checkStreamHandlersInterval = setInterval(() => {
+      const streamHandlersSize = JSON.parse(JSON.stringify((Meteor.StreamerCentral.instances[`cursor-${Auth.meetingID}`].handlers)));
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/cursor/service.js]: This looks expensive. Is there no way to do something like 
`Meteor.StreamerCentral.instances[`cursor-${Auth.meetingID}`].handlers.length`?

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/cursor/service.js]: Screenshot from 2020-01-13 09-44-23

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/cursor/service.js]: Or actually 
Screenshot from 2020-01-13 09-46-55

**Comment by capilkey** [comment]
This PR fixes #8370. It would be really good to get it in before the next 2.2-RC.

**Pull request merged**